### PR TITLE
Add PDF link component

### PR DESCRIPTION
**Pull request by frago12**
## What Changed?

Add PDF link component

**Examples**
```jsx
<Archive title="Download PDF" href="link-to-file.pdf" />
```

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**
- [ ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: src/components/archive.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (gsong) [inline src/components/archive.js:1]: Is this still necessary? I know you no longer need to do this with CRA.

File: src/components/archive.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+export default function Archive({ title, href, ...props }) {
+  return (
+    <Link to={href} title={title} className="small" {...props}>
+      <PdfIcon /> {title}
+    </Link>
+  );
+}
+
+const PdfIcon = ({ className }) => (
```
Comment (gsong) [inline src/components/archive.js:13]: What's the `className` prop about?

File: src/components/archive.js (hunk 1 above)
Comment (frago12) [reply src/components/archive.js:1]: this is Gatsby, so yes, we need the import

File: src/components/archive.js (hunk 2 above)
Comment (frago12) [reply src/components/archive.js:13]: I think it can be removed since no extra styling is needed for this case

Comment (frago12) [reply src/components/archive.js]: removed

**Pull request closed without merging**